go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/CosmWasm/wasmd
 
 go 1.19
 
+// Uses this fork of the wasm08 client so PFM works
+// Now uses github.com/cosmos/ibc-go/modules/capability/types
+// go get github.com/crodriguezvega/ibc-go/v7@24d07389e19357ff836f59a60b2a5d3897baaa08
+replace github.com/cosmos/ibc-go/v7 => github.com/crodriguezvega/ibc-go/v7 v7.0.0-20230626145354-24d07389e193
```

Comment:
why not use https://github.com/cosmos/ibc-go/releases/tag/v7.2.0 ?

Reply:
The wasm-client branch is built off 7.0.0 still right now as it has been in development for a bit. Once their review is done 7.2 will get merged in and bumped to v7.3+. This is a requirement to get the wasm-client logic in for Juno for now for testnet

> https://github.com/strangelove-ventures/ibc-go/tree/feat/wasm-clients-main